src/components/NavBar/MenuDrawer.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import {Button, IconButton, makeStyles, Drawer} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const useStyles = makeStyles({
+   drawer: {
+       width: 250,
+   },
+   link: {
+       color: 'black',
+       textDecoration: 'none',
+   },
+});
+
+function MenuDrawer(props) {
+
+    const classes = useStyles();
+    const [visibility, setVisibility] = useState(false);
+
+    const toggleDrawer = () => {
```

Comment:
can you change toggleDrawer to set visibility to the opposite of what it currently is? see https://stackoverflow.com/a/48394541/11223254. so that instead of `toggleDrawer` and `closeDrawer` we'd only have `toggleDrawer`.